### DISCUSSION TITLE
How to use Discussions

### DISCUSSION THREAD

**Discussion by orta** [DefinitelyTyped/DefinitelyTyped#53392] [_Misc]
## 👋 Welcome!

We’re using Discussions as a place to connect the community of DefinitelyTyped. We hope that you use discussions for:

  * Talking about `@types/xyz` packages
  * Request new `@types` packages

For larger systemic questions/reports, please use [Issues](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/)

**Comment by ajayy-sh** [comment]
How I can import index.d.ts file??
I want contents of that file to be passed to monaco editor
